src/redux/utilities.js

```diff
@@ -0,0 +1,33 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const appID = 'dXhbHW84R2UylfqfKuq7';
+
+export const addBook = createAsyncThunk('Books/addBook', async (book) => {
+  try {
+    await axios.post(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${appID}/books`, book);
+  } catch (e) {
+    throw new Error(e);
+  }
+  return book;
+});
+
+export const deleteBook = createAsyncThunk('Books/deleteBook', async (bookID) => {
+  try {
+    await axios.delete(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${appID}/books/${bookID}`);
+  } catch (e) {
+    throw new Error(e);
+  }
+  return bookID;
+});
+
+export const getBooks = createAsyncThunk('Books/getBooks', async () => {
+  const response = await axios.get(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${appID}/books`);
+  const books = Object.entries(response.data).map((item) => (
```

Comment:
- [ ] Given that I could not add a book, I could not verify if the delete and fetch methods work as required. So kindly ensure they work as well.